core/templates/dev/head/pages/creator_dashboard/creator_dashboard.html

```diff
@@ -38,7 +38,12 @@
           <h1 class="stat-value-with-rating" ng-show="dashboardStats.num_ratings || relativeChangeInTotalPlays"><[dashboardStats.average_ratings || 'N/A']></h1>
           <h1 class="stat-value-without-rating" ng-hide="dashboardStats.num_ratings || relativeChangeInTotalPlays"><[dashboardStats.average_ratings || 'N/A']></h1>
           <p ng-hide="!dashboardStats.num_ratings">
-            (by <[dashboardStats.num_ratings]> users)
+            <p ng-hide="dashboardStats.num_ratings == 1">
+              (by <[dashboardStats.num_ratings]> users)
+            </p>
+            <p ng-hide="dashboardStats.num_ratings > 1">
```

Comment:
Also instead of ng-hide here and above, can you use ng-if? Since that would not render the <p> at all, whereas here, it creates it and hides it.